src/APIBasedAuth.ts

```diff
@@ -373,6 +393,38 @@ declare namespace APIBasedAuth {
     session?: string;
     username?: string;
   };
+
+  export type InitiateSignupResponse = {
+    userConfirmed: boolean;
+  };
+
+  export type InitiateSignupData = {
+    clientId: string;
+    /** Unique username for the user. A required field */
+    username: string;
+    /** Unique email address for the user. A required field */
+    email: string;
+    /**
+     * Web url where the user registration takes place.
+     * @example 'https://example.com'
+     */
+    originalUrl?: string;
+    /**
+     * Password for the user. In the case that a password is not
+     * provided it is assumed that this is a passwordless user and
+     * a password is auto generated for them on the server. The
+     * user is still able to go through the forgot password flow
+     * to set it at a later time
+     */
+    password?: string;
+    /**
+     * An eleven digit phone number for the user
```

Comment:
Super minor, but the phone number format is E.164 structure, if you want to reference it.

Reply:
Awesome! Thank you